python/pyspark/mllib/clustering.py

```diff
@@ -166,12 +166,39 @@ class GaussianMixtureModel(object):
     True
     >>> labels[3]==labels[4]
     True
+    >>> clusterdata_3 = sc.parallelize([[-5.1971], [-2.5359], [-3.8220],
```

Comment:
The data here is different from clusterdata_2 and we can not reshape it, but I can truncate to shorten this.